ansible/playbook/openstack/README.adoc

```diff
@@ -146,11 +146,48 @@ openstack/vm : include_tasks ---------------------------------------------------
 
 === Connect to the new instance
 
-Since all the information related to the host will be managed by our ansible passwordstore roles, which also stores the ssh public and secret keys locally on the `~/.ssh` folder, to login to the newly created VM is as simple as launching the following command.
+All the information related to the host will be managed by our ansible passwordstore roles, which also stores the ssh public and secret keys locally on the `~/.ssh` folder. On the other hand a _bash_ script has been created to make it easier to perform this connection.
 
+To SSH connect to a VM use the `tools/passstore-vm-ssh.sh` bas script.
+
+The 3 arguments to pass to the script are the following.
+
+.Script options
+[%header,cols="2,4"]
+|===
+| Command | Description
+
+| 1: `VM_PROVIDER`
+
+[.fuchsia]#string# / [.red]#required# 
+a| Cloud provider
+
+Choices: 
+
+* `hetzner`
+* `openstack`
```

Comment:
Can we ssh to the IBM cloud VM ?

Reply:
I don't think so, I didn't find a way of doing it.